nexuse2e-webapp/src/main/frontend/src/app/conversation-list/conversation-list.component.ts

```diff
@@ -48,7 +50,7 @@ export class ConversationListComponent implements OnInit {
       allowedValues: ["ERROR", "PROCESSING", "IDLE", "COMPLETED"],
     },
   ];
-  activeFilters: ActiveFilter[] = [];
+  activeFilters: { [fieldName: string]: string | DateRange | undefined } = {};
```

Comment:
Should we use a Map type (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map) instead of a index type?